### DISCUSSION TITLE
This looks awesome! Can it read XLS,Sheets or CSV?

### DISCUSSION THREAD

**Discussion by notagrouch** [AutomaApp/automa#46] [Q&A] [answered]
Hi, this looks great. I saw a few examples and tried my own. I really like this.

But, if I wanted to fill out a form, with repetitive but unique information, like a directory, could I use CSV type of import?

Can it read data from a google sheet or a CSV? if so, how? or do you have an example I could follow?

**Comment by Kholid060** [comment, marked as answer]
This feature is currently in progress, so basically there will be a loop block where it will loop through the data columns or your imported data (JSON or CSV). But, it doesn't support getting data from an online source like google sheet.

**Reply by notagrouch** [reply]
Got it. I think that would work for what I'm thinking about. Is this feature on a roadmap or is there estimated time when it might be working?
